src/core/service.c

```diff
@@ -4068,7 +4070,7 @@ static void service_notify_message(
                         name = NULL;
                 }
 
-                (void) service_add_fd_store_set(s, fds, name);
+                (void) service_add_fd_store_set(s, fds, name, !strv_find(tags, "FDPOLL=0"));
```

Comment:
doesn't matter, but strv_contains() makes this slightly more readable...

Reply:
Eh, changed anyway. :)